<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ogAdjuster</class>
 <widget class="QDialog" name="ogAdjuster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adjust Volume to Hit OG</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Input</string>
       </property>
       <property name="configSection" stdset="0">
        <string notr="true">ogAdjuster</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_sg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>SG</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_sg</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="BtDensityEdit" name="lineEdit_sg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Measured gravity pre-boil</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButton_plato">
          <property name="text">
           <string>Plato</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="BtDensityEdit" name="lineEdit_plato">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Plato (percent by mass of equivalent sucrose)</string>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">plato</string>
          </property>
          <property name="forcedUnit" stdset="0">
           <string notr="true">displayPlato</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButton_temp">
          <property name="text">
           <string>Temp</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="BtTemperatureEdit" name="lineEdit_temp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Temperature of SG reading</string>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">temperature</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="BtTemperatureLabel" name="label_calTemp">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Calibration Temp</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_calTemp</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="BtTemperatureEdit" name="lineEdit_calTemp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Temp to which the hydrometer is calibrated</string>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">calTemp</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="BtVolumeLabel" name="label_volume">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Pre-Boil Volume</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_volume</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="BtVolumeEdit" name="lineEdit_volume">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Measured pre-boil volume</string>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">volume</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="1">
         <widget class="BtDensityEdit" name="lineEdit_og">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>OG if you boil as planned</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_og">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>OG w/o Correction</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="BtVolumeLabel" name="label_add">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Add to Boil</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_add</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="BtVolumeEdit" name="lineEdit_add">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Amount of water you need to add to hit planned OG (or boil off if negative)</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">add</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="BtVolumeLabel" name="label_batchsize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="text">
           <string>Final Batch Size</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_batchSize</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="BtVolumeEdit" name="lineEdit_batchSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Estimated batch size after correction</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="editField" stdset="0">
           <string notr="true">batchsize</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtDensityEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTemperatureEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTemperatureLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtVolumeLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtVolumeEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>label_calTemp</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_calTemp</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_batchsize</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_batchSize</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_add</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_add</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_volume</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_volume</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
